code/scripting/api/objs/debris.cpp

```diff
@@ -11,6 +11,9 @@
 extern int Num_hull_pieces;
 extern debris Hull_debris_list;
 
+extern int Debris_model;
+extern int Debris_vaporize_model;
```

Comment:
Please move the code that needs this into `debris.cpp` to avoid having extern variables.

Reply:
Is externing really a big problem for scripting?  There are two more extern'd variables right above it.

Reply:
I removed all four extern'd variables.